addon/modules/RenameTab.jsm

```diff
@@ -129,12 +132,12 @@ this.RenameTab = {
     var tab = data.tab;
     var label = data.value;
     var resetDefault = aReset || (label == data.docTitle && !data.permanently);
-    var url = resetDefault ? null : data.permanently ? "*" : data.url;
+    var url = resetDefault ? '' : data.permanently ? "*" : data.url;
 
     var win = this.window;
-    win.Tabmix.setItem(tab, "fixed-label", resetDefault ? null : label);
+    win.Tabmix.setItem(tab, "fixed-label", resetDefault ? '' : label);
     win.Tabmix.setItem(tab, "label-uri", url);
-    TabmixSvc.ss.setCustomTabValue(tab, "fixed-label", resetDefault ? null : label);
+    TabmixSvc.ss.setCustomTabValue(tab, "fixed-label", resetDefault ? '' : label);
     TabmixSvc.ss.setCustomTabValue(tab, "label-uri", url);
```

Comment:
we call Tabmix.setItem with null to remove the attribute.

I'v just pushed https://github.com/onemen/TabMixPlus/commit/905c396143d147e4a205e3d55cadfdd7e99202bc
to address this issue with SessionStore.

Just reset your changes to lines 135-141